src/components/SplitSection/styled.ts

```diff
@@ -4,6 +4,7 @@ import { usingColors, usingTypography } from 'src/hooks/theme';
 export const Container = styled.div`
   display: flex;
   flex-direction: column;
+  margin-bottom: 20px;
```

Comment:
Should use typography here instead of a hard-coded value.

Reply:
removed this margin but used `scaleSpacing` on the body padding. What is it for?

Reply:
Two reasons:

- Consistency: we always operate with the same spacing values.
- Maintainability: we can adjust the parameters of typography (or the rythmic scale) to make all spacing of the page smaller or larger (depending on our design) without needing to go through every line of CSS.